packages/parrot-middleware/README.md

```diff
@@ -36,6 +36,19 @@ const scenarios = {
       },
     },
   ],
+  'has more ships with any type': [
+    {
+      request: '/ship_log',
+      response: {
+        type: 'text/plain',
```

Comment:
Should we go with `contentType` instead?

Reply:
I think that's clearer language, I've updated (a379dec)